adamacs/schemas/subject.py

```diff
@@ -97,8 +97,9 @@ class SubjectGenotype(dj.Manual):
     -> Subject
     -> Mutation
     ---
-    genotype        : enum('wt/wt', 'wt/tg', 'tg/wt', 'tg/tg')
+    genotype        : enum('wt/wt', 'wt/tg', 'tg/wt', 'tg/tg', 'cre+', '+/-')
```

Comment:
It's worth considering a separate lookup table for `Genotype`, unless the list of genotypes above is fixed and likely to stay unchanged. Once the table is created, modifying the `enum` to add more entry is not very straightforward.

Reply:
In this case, the two additional items are only to ensure the PyRAT draft works on the test server. Based on previous conversations with Daniel, the four items initially included are the full set of items required for their project, motivating the `enum` data type